price-reporter/src/utils.rs

```diff
@@ -268,9 +274,13 @@ pub fn requires_quote_conversion(exchange: &Exchange) -> bool {
 pub fn parse_pair_info_from_topic(topic: &str) -> Result<PairInfo, ServerError> {
     let parts: Vec<&str> = topic.split('-').collect();
     let exchange = Exchange::from_str(parts[0]).map_err(err_str!(ServerError::InvalidPairInfo))?;
-    let base = Token::from_addr(parts[1]);
-    let quote =
-        if exchange == Exchange::Renegade { Token::usdc() } else { Token::from_addr(parts[2]) };
+    let base = Token::from_addr_without_chain(parts[1]);
+    let chain = base.get_chain();
+    let quote = if exchange == Exchange::Renegade {
+        Token::from_ticker_on_chain(USDC_TICKER, chain)
+    } else {
+        Token::from_addr_on_chain(parts[2], chain)
```

Comment:
continues to require price topics to be constructed with tokens on the same chain